### PR TITLE
layout

### PR DESCRIPTION
**Pull request by dat-hedspi**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -66,29 +72,75 @@ $(document).on('change load', '.question-type', function() {
...
+  /* finish jquery update question */
 
 function remove_fields(link) {
   $(link).prev('input[type=hidden]').val('1');
   $(link).closest('.field').hide();
 }
 
+function checkbox() {
+  var type = $('.question-type').val();
+  console.log(type);
```
Comment (ngtrduc) [inline app/assets/javascripts/application.js]: console log ko can thiet thi xoas di

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -101,7 +153,91 @@ var add = function() {
...
+  $('.signin').css("min-height", y);
+};
+
+$(document).ready(flash);
+$(document).on("page:load", flash);
+$(document).on("page:change", flash);
+
+var scroll_top = function(){
+  $(".circle img").click(function() {
+    $("html, body").animate({ scrollTop: 0 }, "slow");
```
Comment (ngtrduc) [inline app/assets/javascripts/application.js]: convention

File: app/assets/stylesheets/trunk.css [hunk 3]
```diff
@@ -0,0 +1,303 @@
+body {
+	text-rendering: optimizelegibility;
```
Comment (ngtrduc) [inline app/assets/stylesheets/trunk.css]: ident ca file luon

**Comment by tiennh1995** [comment]
ok

**Pull request merged**